.github/workflows/operation-compose-ssl.yml

```
@@ -0,0 +1,69 @@
+name: "Compose: CrateDB with SSL"
+
+on:
+  pull_request:
+    branches: ~
```

actionlint (1.7.4)

character '~' is invalid for branch and tag names. ref name cannot contain spaces, ~, ^, :, [, ?, *. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)